packages/TorneloScoresheet/src/hooks/appMode/arbiterGraphicalRecordingState.ts

```diff
@@ -0,0 +1,40 @@
+import { useContext } from 'react';
+import {
+  AppMode,
+  AppModeState,
+  ArbiterGraphicalRecordingMode,
+} from '../../types/AppModeState';
+import { chessEngine } from '../../chessEngine/chessEngineInterface';
+
+type arbiterGraphicalRecordingStateHookType = [
+  ArbiterGraphicalRecordingMode,
+  {
+    goToRecordingMode: () => void;
+  },
+];
+
+export const makeUseArbiterGraphicalRecordingState =
+  (
+    context: React.Context<
+      [AppModeState, React.Dispatch<React.SetStateAction<AppModeState>>]
+    >,
+  ): (() => arbiterGraphicalRecordingStateHookType | null) =>
+  (): arbiterGraphicalRecordingStateHookType | null => {
+    const [appModeState, setAppModeState] = useContext(context);
+    if (appModeState.mode !== AppMode.ArbiterGraphicalRecording) {
+      return null;
+    }
+
+    const goToRecordingMode = (): void => {
+      const board = chessEngine.fenToBoardPositions(chessEngine.startingFen());
+      setAppModeState({
+        mode: AppMode.GraphicalRecording,
+        pairing: appModeState.pairing,
+        moveHistory: [],
+        board,
+        currentPlayer: appModeState.currentPlayer,
+      });
+    };
```

Comment:
This would reset the game to the starting state (empty move list and starting fen), we want to be able to resume the game...
So we should just copy over the current appModeState, the only thing that changes is the mode, which goes to its respective player mode
ie should be: 
`setAppModeState({ ...appModeState, mode: AppMode.GraphicalRecording });`